src/testing/tree_construction/parser.rs

```diff
@@ -333,6 +334,9 @@ mod tests {
 
     #[test]
     fn parse_data() {
+        let (_, s) = data("#data\n         Test \n#errors\n".into()).unwrap();
```

Comment:
Sometimes when #errors is the last line of a fixture file, this goes wrong. So i'm testing with and without a \n now.